x/interchaintxs/keeper/msg_server.go

```diff
@@ -156,3 +153,25 @@ func (k Keeper) SubmitTx(goCtx context.Context, msg *ictxtypes.MsgSubmitTx) (*ic
 		Channel:    channelID,
 	}, nil
 }
+
+// SerializeCosmosTx serializes a slice of sdk.Msg's using the CosmosTx type. The sdk.Msg's are
+// packed into Any's and inserted into the Messages field of a CosmosTx. The proto marshaled CosmosTx
+// bytes are returned. Only the ProtoCodec is supported for serializing messages.
```

Comment:
Maybe it's better to have a comment describing how and why does this function differ from the original one?